dts/msm8916/msm8916-vivo-y21l.dts

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+
+/ {
+    qcom,msm-id = <0xce 0x00 0xf8 0x00 0xf9 0x00 0xfa 0x00>;
+    qcom,board-id = <0x08 0x0d>;
+
+    model = "Vivo Y21L";
+    compatible = "vivo,y21l", "qcom,msm8916", "lk2nd,device";
```

Comment:
```suggestion
	qcom,msm-id = <0xce 0x00 0xf8 0x00 0xf9 0x00 0xfa 0x00>;
	qcom,board-id = <0x08 0x0d>;

	model = "Vivo Y21L";
	compatible = "vivo,y21l", "qcom,msm8916", "lk2nd,device";
```

Tabs for indents.

Reply:
@wonderfulShrineMaidenOfParadise Tabs only 8 spaces?